### PR TITLE
Adds an env var to enable chaining Cilium plugin.

### PR DESCRIPTION
**Pull request by Weil0ng**
Adds a handle for new env `ENABLE_CILIUM_PLUGIN`, which chains cilium cni at `@cniCiliumPlugin` annotation.

### CODE REVIEW

File: scripts/install-cni.sh [hunk 1]
```diff
@@ -64,6 +64,13 @@ else
 	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
 fi
 
+if [ "${ENABLE_CILIUM_PLUGIN}" == "true" ]
+then
+        cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniCiliumPlugin#,{\"type\": \"cilium-cni\"}#g")
```
Comment (cadmuxe) [inline scripts/install-cni.sh]: cloud you also update the netd.yaml, we are using it as a reference. 

https://github.com/GoogleCloudPlatform/netd/blob/master/netd.yaml#L71

Comment (Weil0ng) [reply scripts/install-cni.sh]: Included another commit for this :)

**Comment by Weil0ng** [comment]
PTAL :) @cadmuxe

**Pull request merged**